Generating DNSSEC keys through the API results in internal server error

POST api/1.2/cdns/dnsseckeys/generate returns the following:

[2017-09-28 20:54:26,318] [ERROR] Can't locate object method "create" via package "Net::DNS::RR::DS" (perhaps you forgot to load "Net::DNS::RR::DS"?) at /opt/traffic_ops/app/lib/MojoPlugins/DnssecKeys.pm line 249.

--- a/traffic_ops/app/lib/MojoPlugins/DnssecKeys.pm
+++ b/traffic_ops/app/lib/MojoPlugins/DnssecKeys.pm
@@ -23,6 +23,7 @@ use Crypt::OpenSSL::RSA;
 use Crypt::OpenSSL::Bignum;
 use Crypt::OpenSSL::Random;
 use Net::DNS::SEC::Private;
+use Net::DNS::RR::DS;
 use Data::Dumper;
 use JSON;
 my $TMP_LOCATION = "/var/tmp";

